integration/runner/runner.go

```diff
@@ -78,23 +78,29 @@ func RunCommand(cmd string, args ...string) error {
 	return nil
 }
 
+func RunSshCommand(cmd string, args ...string) error {
+	if *sshOptions != "" {
+		args = append(strings.Split(*sshOptions, " "), args...)
+	}
+	return RunCommand(cmd, args...)
+}
+
 func PushAndRunTests(host, testDir string) error {
 	// Push binary.
 	glog.Infof("Pushing cAdvisor binary to %q...", host)
-	args := common.GetGCComputeArgs("ssh", host, "--", "mkdir", "-p", testDir)
-	err := RunCommand("gcloud", args...)
+
+	err := RunSshCommand("ssh", host, "--", "mkdir", "-p", testDir)
 	if err != nil {
 		return fmt.Errorf("failed to make remote testing directory: %v", err)
 	}
 	defer func() {
-		args := common.GetGCComputeArgs("ssh", host, "--", "rm", "-rf", testDir)
-		err := RunCommand("gcloud", args...)
+		err = RunSshCommand("ssh", host, "--", "rm", "-rf", testDir)
 		if err != nil {
 			glog.Errorf("Failed to cleanup test directory: %v", err)
 		}
 	}()
-	args = common.GetGCComputeArgs("copy-files", cadvisorBinary, fmt.Sprintf("%s:%s", host, testDir))
-	err = RunCommand("gcloud", args...)
+
+	err = RunSshCommand("scp", "-r", cadvisorBinary, fmt.Sprintf("%s:%s", host, testDir))
```

Comment:
I think scp takes different options from ssh - maybe this should just be RunCommand?